### PR TITLE
Replace dasen() with adjustedDasen() function

### PR DESCRIPTION
**Pull request by alicemfr**
# Description

This pull request will change the normalisation method from dasen() to adjustedDasen() (line 117 and 132). The latter method separates autosomal DNA methylation sites from those on the sex chromosomes to avoid introducing an artificial sex bias in normalised betas values.

## Issue ticket number

This pull request is to address issue: #210

## Type of pull request

- [ ] Bug fix
- [x] New feature/enhancement
- [ ] Code refactor
- [ ] Documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my code to check that it is functional
- [ ] I have used linters to check for common sources of errors
- [ ] I have implemented fail safes in my code to account for edge cases
- [ ] I have made the corresponding changes to the documentation

### CODE REVIEW

**Comment by sof202** [comment]
The old `dasen` is still used in the following places:

```
array/DNAm/preprocessing/calcQCMetrics.r|283 col 3| dasen(gfile, node="normbeta")
array/DNAm/analysis/methodsDevelopment/compareNormalisation.r|76 col 11| normbeta<-dasen(meth, unmeth, probeAnnot$designType)
```

File: array/DNAm/preprocessing/normalisation.r [hunk 1]
```diff
@@ -1,150 +1,158 @@
...
+
+#----------------------------------------------------------------------#
+# NORMALISE ALL SAMPLES TOGETHER
+#----------------------------------------------------------------------#
+
+normbeta<-adjustedDasen(
+			onetwo = probeAnnot$designType,
+			chr = probeAnnot$CHR,
+			mns = meth,
+			uns = unmeth)
```
Comment (sof202) [inline array/DNAm/preprocessing/normalisation.r:121]: This is the correct usage of the function according to wateRmelon's man page

**Comment by alicemfr** [comment]
> array/DNAm/preprocessing/calcQCMetrics.r|283 col 3| dasen(gfile, node="normbeta")
Yes good point. @ew367 do you know why we do adjustedDasen() for specifically EPICv2 here?

**Comment by ew367** [comment]
> > array/DNAm/preprocessing/calcQCMetrics.r|283 col 3| dasen(gfile, node="normbeta")
> > Yes good point. @ew367 do you know why we do adjustedDasen() for specifically EPICv2 here?

@alicemfr its legacy code and needs updating so that all array types use adjustedDasen for the calcQCmetrics script as well as the normalisation script

**Comment by alicemfr** [comment]
@ejh243 This file uses dasen, not adjustedDasen. Do you think it needs updating or just keep adjustedDasen to within the QC scripts?
`array/DNAm/analysis/methodsDevelopment/compareNormalisation.r`

**Review by sof202 — APPROVED** [review]

**Comment by sof202** [comment]
> @ejh243 This file uses dasen, not adjustedDasen. Do you think it needs updating or just keep adjustedDasen to within the QC scripts? `array/DNAm/analysis/methodsDevelopment/compareNormalisation.r`

This can be left for a different PR, the main purpose of this one was to change the QC pipeline.

**Pull request merged**